src/includes/sourcemaps/best-practices/javascript.mdx

```diff
@@ -0,0 +1,158 @@
+## A Simple Setup
+
+Here is a simple project where your minified / transpiled files and their source maps reside in the same directory like below:
+
+```{tabTitle: Project Structure}
+├── build/
+│   ├── worker.js
+│   ├── worker.js.map
+│   ├── app.js
+│   ├── app.js.map
+│   ├── index.html
+├── package.json
+├── public/
+│   └── index.html
+├── sentry.properties
+├── src/
+│   ├── app.js
+│   └── worker.js
+├── webpack.config.js
+```
+
+For this project, can use a simple Sentry configuration like:
+
+```javascript {tabTitle: webpack.config.js}
+// ...
+new SentryWebpackPlugin({
+    // sentry-cli configuration
+  authToken: process.env.SENTRY_AUTH_TOKEN,
+  org: "___ORG_SLUG___",
+  project: "___PROJECT_SLUG___",
+  // webpack specific configuration
+  include: "build",
+  configFile: "sentry.properties",
+  release: process.env.SENTRY_RELEASE,
+}),
+// ...
+```
+
+## Selecting Tool
+
+`sentry-webpack-plugin` is the recommended way to integrate your source maps into Sentry. You can use `sentry-cli` if you are not using webpack in your project.
```

Comment:
@PeloWriter, The second part of this is for projects that do not use Webpack at all, so updating to `Webpack`, instead of `the Webpack plugin` if it's ok?